OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -25,6 +26,7 @@ public class FrozenActorLayerInfo : ITraitInfo
 	public class FrozenActor
 	{
 		public readonly MPos[] Footprint;
+		public readonly CPos[] OccCells;
```

Comment:
Yes, that's for Occupied Cells. But there already is a method called OccupiedCells() so I wanted to give this a different name to avoid confusion.